go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cosmtrek/air
 
-go 1.21
+go 1.22.0
```

Comment:
Probably want to keep this on the major version?

Reply:
Initially tried this but received the following error on `go mod tidy`

```
go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available
```

Reply:
https://github.com/golang/go/issues/62278#issuecomment-1693538776

Reply:
Fixed. Was a local installation issue.